src/main/java/com/minecolonies/client/gui/WindowBuildTool.java

```diff
@@ -448,8 +447,20 @@ private void updateLevelButton()
      */
     private void confirmClicked(Button button)
     {
-        MineColonies.getNetwork().sendToServer(new BuildToolPlaceMessage(hutDec.get(hutDecIndex),
-                                                                          getStyles().get(styleIndex), this.pos, rotation, Settings.instance.isInHutMode()));
+        InventoryPlayer inventory = this.mc.thePlayer.inventory;
```

Comment:
Would declaring this as final be valid?  The code would be client side so it should not be an issue for SMP; but does the players inventory object ever change in the event of death or other mods?